app/assets/stylesheets/articles.scss

```diff
@@ -1564,10 +1529,6 @@
           opacity: 0.9;
         }
       }
-      img {
```

Comment:
Pulled this out as it was breaking sidebar widget styles for the icons, e.g. Reading List icon. I checked and these styles are not being applied anywhere. Social icons and sponsors have their own CSS classes as do the Reading List icon etc. And in those classes, the border is set to none.